sentinelsat/sentinel.py

```diff
@@ -164,7 +164,7 @@ def format_query(area=None, date=None, raw=None, area_relation='Intersects',
             # Escape spaces, where appropriate
             if isinstance(value, string_types):
                 value = value.strip()
-                if not any(value.startswith(s[0]) and value.endswith(s[1]) for s in ['[]', '{}', '//']):
+                if not any(value.startswith(s[0]) and value.endswith(s[1]) for s in ['[]', '{}', '//', '()']):
```

Comment:
I'm not sure why I did not include `()` in this list. This should be absolutely fine.